interface/patient_tracker/patient_tracker.php

```diff
@@ -36,6 +36,24 @@
 require_once("$srcdir/options.inc.php");
 require_once("$srcdir/patient_tracker.inc.php");
 
+#define variables, future enhancement allow changing the to_date and from_date 
+#to allow picking a date to review
+ 
+$appointments = array();
+$from_date = date("Y-m-d");
+$to_date = date("Y-m-d");
+$datetime = date("Y-m-d H:i:s");
+
+# go get the information and process it
+$appointments = fetch_Patient_Tracker_Events($from_date, $to_date);
+$appointments = sortAppointments( $appointments, 'time' );
+
+$chk_prov = array();  // list of providers with appointments
+
+// Scan appointments for additional info
+foreach ( $appointments as $apt ) {
+  $chk_prov['uprovider_id'] = $apt['ulname'] . ', ' . $apt['ufname'] . ' ' . $apt['umname'];
```

Comment:
I think that no matter how many providers you have, this logic will always just count to 1 when do count($chk_prov). 
Maybe something like:
``````
$chk_prov_all[] = $apt['ulname'] . ', ' . $apt['ufname'] . ' ' . $apt['umname'];
$chk_prov_count = count(array_count_values($chk_prov_all));